restaurants/restaurants.json

```diff
@@ -1,80 +0,0 @@
-[{
-    "id": "pitcher-inn",
-    "name": "The Pitcher Inn & Restaurant",
-    "address":"275 Main, Warren, VT 05674",
-    "hours": "4:00 pm - 1:00 am",
-    "latitude": "44.114933",
-    "longitude": "-72.854628",
-    "phone number": "(802)496-6350",
-    "notes": [
-        "Upscale food and ambience, you can choose white-linen dinner service or a more relaxed bar atmosphere in Tracks Bar.",
-        "Great place to bring a date or go out for an occasion, say hi to Lucas at the bar."
-    ]
-},
```

Comment:
Having both a restaurants.json with every restaurant in it, and an individual restaurant.json file for each one is a bit redundant. you can either do a fetch for all of them and parse it. or you could do a fetch to each one individually when needed.